lib/lotus/helpers/html_helper.rb

```diff
@@ -184,8 +193,13 @@ module HtmlHelper
       #
       # @see Lotus::Helpers::HtmlHelper
       # @see Lotus::Helpers::HtmlHelper::HtmlBuilder
-      def html
-        HtmlBuilder.new
+      # @see Lotus::Helpers::HtmlHelper::HtmlFragment
+      def html(&blk)
+        if block_given?
+          HtmlBuilder.new.fragment(&blk)
```

Comment:
@huynhquancam Never mind again :sweat_smile: 

Let's the production code as it is, but please add the other tests and fix the docs, then it's mergeable! Thank you very much! :+1: